symplyphysics/laws/electricity/circuits/attenuation_coefficient_in_metal_in_rectangular_waveguide_for_transverse_magnetic_waves.py

```diff
@@ -0,0 +1,69 @@
+from sympy import (Eq, solve, sqrt,)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless,)
+
+
+# Description
+## A coaxial waveguide is an electrical cable consisting of a central conductor and a shield arranged coaxially and separated
+## by an insulating material or an air gap. It is used to transmit radio frequency electrical signals.
+## The specific resistance of a coaxial waveguide depends on the radius of the outer conductor and the radius of the inner conductor,
+## as well as on the relative permeability of the insulator material, frequency of signal and specific conductivity of conductor.
+## The attenuation coefficient shows how many times the transmitted signal weakens per unit length of the coaxial waveguide.
+
+## Law is: am = 2 * Rs * (n^2 * (b / a)^3 + m^2) / (Z0 * b * sqrt((1 - (L / L1)^2) * (n^2 * (b / a)^2 + m^2))), where
```

Comment:
Show the link to the law and test data if possible. There are many ways to make a mistake in this formula so I want to double check it.

Reply:
> Show the link to the law and test data if possible. There are many ways to make a mistake in this formula so I want to double check it.

This is a reference to the law. Page 8.
https://core.ac.uk/download/pdf/49213156.pdf

But I can't find a test case.

Reply:
I do not see anything related on page 8. Screenshot maybe?

Reply:
> I do not see anything related on page 8. Screenshot maybe?

I sent the wrong link. Page 8.
http://old.istu.ru/files/material-static/1664/glava07.pdf

![image](https://github.com/blackyblack/symplyphysics/assets/100017508/37d0a401-ab5b-4ac4-92d8-3e5733e9b148)

Reply:
Well, it does not correspond to the code you wrote.

Reply:
> Well, it does not correspond to the code you wrote.

Yes, I have already noticed and corrected it.

Reply:
Still does not correspond to description.